corgi/tasks/sca.py

```diff
@@ -83,12 +83,13 @@ def save_component(component: dict[str, Any], parent: ComponentNode) -> bool:
     name = meta.pop("name", "")
     try:
         # Use all fields from Component index and uniqueness constraint
+        # Don't update_or_create since Syft's metadata shouldn't override Brew's
         obj, created = Component.objects.get_or_create(
             type=component["type"],
             name=name,
             version=meta.pop("version", ""),
             release="",
-            arch="",
+            arch="noarch",
```

Comment:
Looks sane to me. Although your question got me thinking if we are setting arch correctly for RPM components detected by Syft. I guess setting arch to "noarch" instead of "" is an improvment so I'm happy to ahead with this PR as is and revisit the arch for RPMs detected by Syft later.